stkSWIV.sol

```diff
@@ -6,59 +6,113 @@ import './Utils/SafeTransferLib.sol';
 
 contract stkSWIV is ERC20 {
 
-    ERC20 immutable SWIV;
+    ERC20 public immutable SWIV;
```

Comment:
```suggestion
    address public immutable SWIV;
```
As a universal pattern, I think we try define these attributes as `address` types and then cast them to their interfaces when we use them.